pkg/server/helpers.go

```diff
@@ -349,3 +352,38 @@ func getUserFromImageUser(user string) (*int64, string) {
 	// If user is a numeric uid.
 	return &uid, ""
 }
+
+// getImageMetadata returns corresponding metadata of the image reference, if image is not
```

Comment:
pulling the image makes getting metadata for the image expensive...  Is there a use case for getting metadata for images that have not yet been pulled for containers?